### PR TITLE
Use better assertions for computedrole

### LINKED ISSUE #39118: Some role assertions in webdriver/tests/get_computed_role/get.py seem incorrect
Some [role assertions in webdriver/tests/get_computed_role/get.py](https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/get_computed_role/get.py#L79) seem incorrect.

```python
@pytest.mark.parametrize("html,tag,expected", [
    ("<li role=menuitem>foo", "li", "menuitem"),
    ("<input role=searchbox>", "input", "searchbox"),
    ("<img role=presentation>", "img", "presentation")])
```

1. `menuitem` is orphaned, not in a menu, so this test likely asserts a shared browser bug. (WK issue is possibly [130364](https://webkit.org/b/130364)). Suggestion: `("<main>foo</main>", "main", "main"),`
2. `searchbox` appears correct.
3. `presentation` asserts a synonym of `none` which is not specified (discussed in [Core-AAM 166](https://github.com/w3c/core-aam/issues/166)) and results differ between browsers. This test could change to something less ambiguous like `button`… Suggestion: `("", "img", "button")])`

### PR DESCRIPTION
**Pull request by cookiecrook**
Resolves #39118

Replaces two invalid computedrole assertions which relied on an orphaned menuitem and a previously unspecified but now incorrect role synonym. 

All three of the updated computedrole tests pass in WebKit and Chromium ( and Gecko soon), and this test effectively exercises the webdriver computedrole getter. Other disputed or erroneous tests that were removed here will be handled in /html-aam and /wai-aria/role

More detail in #39118

### CODE REVIEW

**Comment by cookiecrook** [comment]
@jcsteh looks like Gecko is [failing the new "main" role test](https://wpt.fyi/results/webdriver/tests/get_computed_role/get.py?diff&filter=ADC&run_id=5200271116861440&run_id=5075676565078016). Would you like to suggest another innocuous one for this webdriver test, since that specific role failure is already tested in /html-aam/role/roles.html?

**Comment by jcsteh** [comment]
ComputedRole is currently pretty broken in Gecko, as it exposes our internal role strings rather than standardised ARIA roles. I've been working on this and I have a bunch of patches up for review to fix it. Once those land, this test should pass.

That said, `<article>foo</article>` should pass even with the current brokenness.

**Comment by cookiecrook** [comment]
Gecko fails for `<nav>` too, but I have not been able to get WPT running in Firefox so I'm not going to play trial and error with the Azure Pipeline. I'll add an `<h2>` heading as the last attempt, but if that doesn't pass, I'll leave as-is. These are all valid failures in Gecko.

**Comment by cookiecrook** [comment]
Confirmed Gecko passes the `<article>` test. Thanks Jamie.

Okay to merge once @patrickangle or @AutomatedTester reviews.

File: webdriver/tests/get_computed_role/get.py
Comment (juliandescottes) [inline webdriver/tests/get_computed_role/get.py:82]: Thanks for the patch! On Firefox side, I had filed https://bugzilla.mozilla.org/show_bug.cgi?id=1828600 to review this test, and I was wondering if the two following test cases would make sense: 

> ("", "img", "img"),
Check that an invalid role leads to return the default role for the element

> ("", "img", None),
Check that presentation/none effectively remove the element from the accessibility tree

Do you think they would be worth adding? I only checked that they passed with the recent work from @jcsteh , so I don't know if that would work for other vendors?

Comment (jcsteh) [reply webdriver/tests/get_computed_role/get.py:82]: This null vs "" vs "none" thing is being discussed in https://github.com/web-platform-tests/wpt/pull/39618#pullrequestreview-1393840907 also. I probably need to file a separate issue for it.

**Comment by jgraham** [comment]
This doesn't require review from all the people marked as reviewers. In particular if the people working on a11y are happy with this change, and it doesn't also require a WebDriver spec change, we should land it without review from a WebDriver reviewer (if it _does_ require a spec change we could still find a way to land it as a tentative test without WebDriver review, as long as there's a spec issue open).

**Review by juliandescottes — APPROVED** [review]
Given the conversation, I think it's fine to approve this, as long as @jcsteh is happy with the change as well.

File: webdriver/tests/get_computed_role/get.py
Comment (cookiecrook) [reply webdriver/tests/get_computed_role/get.py:82]: There are also a lot of tests in /wai-aria and /html-aam that will more fully validate this accessibility parts without resulting in spurious failures to this WebDriver getter test.

**Pull request merged**
